testsuite/cli/src/client_proxy.rs

```diff
@@ -453,12 +453,43 @@ impl ClientProxy {
 
         ensure!(num_coins > 0, "Invalid number of coins to mint.");
 
+        if self.assoc_root_account.is_some() {
+            let script = transaction_builder::encode_create_testing_account_script(
+                type_tag_for_currency_code(currency_code.clone()),
+                receiver,
+                receiver_auth_key.prefix().to_vec(),
+                false, /* add all currencies */
+            );
+            // If the account is local, then create the account now.
+            if let Some(pos) = self
+                .accounts
+                .iter()
+                .position(|account_data| account_data.address == receiver)
+            {
+                let status = &self.accounts.get(pos).unwrap().status;
```

Comment:
Is it possible to do `get_mut` here and avoid the second unwrap at 476?

Reply:
Sadly it's not :( the Rust borrowchecker complains (rightly) in this case that `self` is mutably borrowed on line 472 to send the transaction